packages/editor-ui/src/components/ResourceLocator/ResourceLocator.vue

```diff
@@ -1,12 +1,12 @@
 <template>
 	<div
-		ref="container"
+		ref="containerRef"
 		class="resource-locator"
 		:data-test-id="`resource-locator-${parameter.name}`"
 	>
 		<OnClickOutside @trigger="hideResourceDropdown">
```

Comment:
Nit: Since this is now migrated, we can use `onClickOutside` as a function and get rid of the additional component.